src/stackdriver-nozzle/nozzle/parse_events.go

```diff
@@ -0,0 +1,47 @@
+package nozzle
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/cloudfoundry/sonde-go/events"
+)
+
+func ParseEvents(names []string) ([]events.Envelope_EventType, error) {
+	events := []events.Envelope_EventType{}
+
+	for _, name := range names {
+		if name == "" {
```

Comment:
This would allow the config to contains something like `foo,,bar`. Do we want that, or should it error instead of continuing?

Reply:
That's a good point. I don't think we want to explicitly allow that. The alternate here is to special case empty string [here](https://github.com/cloudfoundry-community/stackdriver-tools/pull/117/files#diff-f484551b1a038698c9d7f1840e368a1cR90) (or move the parsing into the config object).

The reason I added this is because: `strings.Split("", ",") => []string{""}`